File: OOP with C#/Homework 3/RouteCipher/RouteCipher/RouteCipherTest.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RouteCipher
{
    public class RouteCipherTest
    {
        public static void Main(string[] args)
        {
            RouteCipher routeCipherOne = new RouteCipher(-5);
            Console.WriteLine(routeCipherOne.Encrypt("abort the mission, you have been spotted"));
            Console.WriteLine(routeCipherOne.Decrypt("&teanitrobatsyvnted&oehomehsoespbui"));
            Console.WriteLine(routeCipherOne.ToString());

            Console.WriteLine();

            RouteCipher routeCipherTwo = new RouteCipher(4);
            Console.WriteLine(routeCipherTwo.Decrypt("TIEIXT&&EAHSIHSPNTLT"));
            Console.WriteLine(routeCipherTwo.Encrypt("THISISTHEPLAINTEXT"));
            Console.WriteLine(routeCipherTwo.ToString());

            Console.WriteLine();

            RouteCipher routeCipherThree = new RouteCipher(2);
            Console.WriteLine(routeCipherThree.Encrypt("Date: 01.11.2021"));
            Console.WriteLine(routeCipherThree.Decrypt("Dt01221011ea"));
            Console.WriteLine(routeCipherThree.ToString());
        }
    }
}

File: OOP with Java/Homework 4/src/RouteCipher/RouteCipherTest.java
package RouteCipher;

public class RouteCipherTest {
    public static void main(String[] args) {
        RouteCipher cipherOne = new RouteCipher(-5);
        String cipherTextOne = "abort the mission, you have been spotted", encryptedTextOne = "", decryptedTextOne = "";
        encryptedTextOne = cipherOne.encrypt(cipherTextOne);
        decryptedTextOne = cipherOne.decrypt(encryptedTextOne);
        System.out.printf("Original text: %s \nEncrypted Text: %s \nDecryptedText: %s", cipherTextOne, encryptedTextOne, decryptedTextOne);

        RouteCipher cipherTwo = new RouteCipher(4);
        String cipherTextTwo = "This is the plain text.", encryptedTextTwo = "", decryptedTextTwo = "";
        encryptedTextTwo = cipherTwo.encrypt(cipherTextTwo);
        decryptedTextTwo = cipherTwo.decrypt(encryptedTextTwo);
        System.out.printf("\n\nOriginal text: %s \nEncrypted Text: %s \nDecryptedText: %s", cipherTextTwo, encryptedTextTwo, decryptedTextTwo);
    }
}
